config/deploy.rb

```diff
@@ -46,6 +46,7 @@
   task :wp_permissions do
     on roles(:app) do
       execute :chmod, "644 #{release_path}/public/wp-config.php"
+      execute :chmod, "-R 755 #{release_path}/content/uploads"
```

Comment:
Aren't files usually `644`? This appears to make everything `755`. 

Do we have a reference for this? I'd like to see a code comment that assures us we based these settings on a reliable source.